GoogleCloudPlatform/bank-of-anthos
# Used a ThreadPoolExecutor to make concurrent API calls

### Fixes #353 

### Background 
Currently the fronted makes serial calls to the backend services, with this change it will make concurrent calls and therefore return the page quicker.

### Change Summary
Change from serial to concurrent calls to the backend services.

### Testing Procedure
Deploy the application in an environment and check the loading times for the home page.

### Related PRs or Issues 
None
